rachellegelden/Home-Together-Canada
# HTC-317: Search Criteria Front-End

# [Issue 317](https://github.com/rachellegelden/Home-Together-Canada/issues/317)

closes #317 
closes #320 
closes #321 

## Summary
- Created front end components for search criteria
- Added CSS 
- Added Validation
- Added invalid CSS 
- Added Registration Request Data

## Relevant Motivation & Context
Member Sign Up

## Testing Instructions
Navigate to Member Registration. Ensure all fields are present, all variables are assigned correctly and validation works as excepted 

## Developer checklist prior to opening this pull request:


## Reviewer
